js/data.js

```diff
@@ -29,3 +35,35 @@ var responses = {
 
 // eslint-disable-next-line no-unused-vars
 var typeCheck = ['arcsin', 'arccos', 'arctan', 'sin', 'cos', 'tan', 'log'];
+
+// eslint-disable-next-line no-unused-vars
+var trigProblemTypes = ['sin', 'cos'];
+
+// eslint-disable-next-line no-unused-vars
+var practiceTypes = ['simplify', 'factor', 'derive', 'integrate'];
+
+// eslint-disable-next-line no-unused-vars
+var derivativeTypes = ['poly', 'trig', 'trigExponent'];
+
+// eslint-disable-next-line no-unused-vars
+var integralTypes = ['poly', 'trig', 'exponent'];
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Instead of disabling `no-unused-vars`, you can add this to the top of your file
```js
/* exported typeCheck, trigProblemTypes, practiceTypes, etc, etc */
```